lib/internal/util.js

```diff
@@ -506,8 +507,12 @@ function isInsideNodeModules() {
       const filename = frame.getFileName();
       // If a filename does not start with / or contain \,
       // it's likely from Node.js core.
-      if (RegExpPrototypeExec(/^\/|\\/, filename) === null)
+      if (
+        filename[0] !== '/' &&
+        StringPrototypeIncludes(filename, '\\') === false
```

Comment:
This only applies to Windows right?

Reply:
this PR is already old but afaik it was effecting linux too